include/deal.II/base/memory_space.h

```diff
@@ -19,17 +19,25 @@
 
 #include <deal.II/base/config.h>
 
+#ifdef DEAL_II_USE_KOKKOS_BACKEND
+#  include <Kokkos_Core.hpp>
+#endif
+
 DEAL_II_NAMESPACE_OPEN
 
 /**
  */
 namespace MemorySpace
 {
+#ifdef DEAL_II_USE_KOKKOS_BACKEND
+  using Host = ::Kokkos::HostSpace;
+#else
```

Comment:
You will want to copy the documentation of the structure here as well so that it shows up in doxygen.